src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Text.Json
+{
+    public enum JsonCommentHandling : byte
+    {
+        AllowComments = (byte)1,
+        Default = (byte)0,
+        SkipComments = (byte)2,
+    }
+    public partial class JsonReaderException : System.Exception
+    {
+        public JsonReaderException(string message, long lineNumber, long lineBytePosition) { }
+        public long LineBytePosition { get { throw null; } }
```

Comment:
Maybe "BytePositionInLine"?  "LineBytePosition" could be interpreted as either the byte position at which the line starts or as the byte position in the current line.